test/AutoRest.TestServerLowLevel.Tests/LowLevel/Generation/ModelWriterTests.cs

```diff
@@ -190,7 +190,7 @@ public PrimitivePropertyModel(string requiredString,int requiredInt,long require
 
 namespace Cadl.TestServer.PrimitiveProperties
 {
-public partial class PrimitivePropertyModel : global::Azure.Core.IUtf8JsonSerializable
+public partial class PrimitivePropertyModel: global::Azure.Core.IUtf8JsonSerializable
 {
 void global::Azure.Core.IUtf8JsonSerializable.Write(global::System.Text.Json.Utf8JsonWriter writer)
```

Comment:
These are example snippets? Looks like our simplifier cannot simplify things in the xml document, would this be a problem?

Reply:
No, this hasn't gone through the simplifier, we do comparison straight out of the writer.